package.json

```diff
@@ -12,21 +12,17 @@
   },
   "author": "YScope Inc.",
   "peerDependencies": {
-    "@stylistic/eslint-plugin-js": "^1.6.2",
-    "@stylistic/eslint-plugin-jsx": "^1.6.2",
-    "@stylistic/eslint-plugin-plus": "^1.6.2",
-    "eslint": "^8.57.0",
-    "eslint-plugin-import": "^2.29.1",
+    "@stylistic/eslint-plugin": "^2.12.1",
+    "eslint": "^9.17.0",
+    "eslint-import-resolver-typescript": "^3.7.0",
+    "eslint-plugin-import": "^2.31.0",
     "eslint-plugin-import-newlines": "^1.4.0",
-    "eslint-plugin-jsdoc": "^48.2.3",
-    "eslint-plugin-no-autofix": "^1.2.3",
-    "eslint-plugin-react": "^7.33.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-simple-import-sort": "^12.0.0"
-  },
-  "eslintConfig": {
-    "extends": [
-      "./common"
-    ]
+    "eslint-plugin-jsdoc": "^50.6.1",
+    "eslint-plugin-no-autofix": "^2.1.0",
+    "eslint-plugin-react": "^7.37.3",
+    "eslint-plugin-react-hooks": "^5.1.0",
+    "eslint-plugin-simple-import-sort": "^12.1.1",
+    "globals": "^15.14.0",
+    "typescript-eslint": "^8.18.2"
```

Comment:
New version available.

Reply:
(not directly related to this comment)

How do you feel about enabling Dependabot version updates in our web repos? Reference: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference

Reply:
I've enabled all the code security settings for all public repos. If we need to customize them for a particular repo, we can use the local dependabot file, but I guess we won't need to any time soon.

Reply:
Right, enabling the code security Dependabot is good enough for receiving PRs for the dependencies that contains security vulnerabilities, but in order to get updates for other dependencies, I believe the local dependabot file is the only missing piece. 

With that said, whether we want to keep all our dependencies up to date can be quite opinionated though, lol

Reply:
Oh, I see. Sure, I'm not opposed to enabling that. The way I see it, dependabot serves as a notifier and we'd still have the option to reject the update if we have concerns.